dev-libs/json-c/metadata.xml

```diff
@@ -18,10 +18,10 @@
   </use>
   <longdescription lang="en">
 "A JSON implementation in C" is probably the better description, and then
-"JSON-C implements a reference counting object model that allows you to 
-easily construct JSON objects in C, output them as JSON formatted 
-strings and parse JSON formatted strings back into the C 
-representation of JSON objects.
+"JSON-C implements a reference counting object model that allows you to
+easily construct JSON objects in C, output them as JSON formatted
+strings and parse JSON formatted strings back into the C
+representation of JSON objects".
```

Comment:
Overall this looks more like a comment than a proper description. Can we make it clearer, or remove it altogether?

Reply:
This message is taken almost verbatim from https://github.com/json-c/json-c. But, yes, it reads a bit weird. I will adjust it a little.